js/app.js

```diff
@@ -26,74 +26,38 @@ function Item(name, filepath, id){
 //using for loop create 20 objects
 for(var i = 0; i < stockImages.length; i++){
   var filepath = 'img/' + stockImages[i] + '.jpg';
-  console.log(filepath);
   new Item(stockImages[i], filepath, i);
-  console.log(objStore);
 }
-console.log('objects are created');
-console.log(Item);
 
+function pickInitialNum(){
+  isSame = false;
+  while(isSame === false){
```

Comment:
Since `isSame` is a boolean, you could just use the condition `!isSame` to loop while `isSame` is `false`.